core/src/main/java/org/apache/carbondata/core/mutate/CarbonUpdateUtil.java

```diff
@@ -471,6 +471,13 @@ public static void cleanUpDeltaFiles(CarbonTable table, boolean forceDelete) thr
     LoadMetadataDetails[] details =
         SegmentStatusManager.readLoadMetadata(table.getMetadataPath());
 
+    SegmentUpdateStatusManager updateStatusManager = new SegmentUpdateStatusManager(table);
+    SegmentUpdateDetails[] segmentUpdateDetails = updateStatusManager.getUpdateStatusDetails();
```

Comment:
Please add comments why this change is required

Reply:
@Indhumathi27 added